### ISSUE TITLE
Root description never displayed.

### ISSUE THREAD

**Issue by cdibbs** [vvakame/commandpost#4]
I have been hard-pressed to get the text passed into description to display:

```typescript
            let root = commandpost
                .create<any, any>("")
                .version(this.pkg.version, "-v, --version")
                .description("This is a fancy program that does neat things.");
               // ...
            commandpost
                .exec(root, this.process.argv);
```

Am I doing something wrong? Thanks!

**Comment by vvakame** [comment]
sorry for late reply.
I was fix it and ship soon. please wait.

**Comment by cdibbs** [comment]
@vvakame No problem. Thanks for the fix!

**Comment by vvakame** [comment]
I just released v1.1.0. please check it. 😉
